app/views/handlers/question.py

```diff
@@ -155,3 +155,13 @@ def evaluate_and_update_section_status_on_list_change(self, list_name):
                 section_id=section_id,
                 list_item_id=list_item_id,
             )
+
+    def clear_radio_answers(self):
+        answer_ids_to_remove = []
+        for answer in self.rendered_block.get("question").get("answers"):
```

Comment:
The `question` and `answers` are always a valid key right?. As in do we need `.get()`?

Reply:
Fair point, removed the `.get`s